pyproject.toml

```diff
@@ -77,6 +77,18 @@ exclude = ["marin/", "scripts/"]
 [tool.pyrefly]
 project-includes = ["lib/marin/src/**/*.py", "lib/levanter/src/**/*.py"]
 
+# Explicitly tell Pyrefly where our editable packages live so it resolves imports
```

Comment:
Oh nice, I thought I had added this before, might have gotten lost in the shuffle.